source/components/Autocomplete.js

```diff
@@ -64,9 +64,16 @@ class Autocomplete extends Component {
     themeOverrides: {}
   };
 
-  constructor(props) {
+  constructor(props: Props) {
     super(props);
 
+    // $FlowFixMe
+    this.rootElement = React.createRef();
+    // $FlowFixMe
+    this.inputElement = React.createRef();
+    // $FlowFixMe
+    this.suggestionsElement = React.createRef();
```

Comment:
More context about the `$FlowFixMe` would be great here!

Reply:
Flow throws an error saying that the createRef method can't be found on the React module. This is because flow hasn't been updated with support for React v16.3^ yet. I checked on this and it won't be long before it's updated. This also happens on types imported from React like SynthenticInputEvent.